testkit/package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
A small hack to make the testkits available under

`import {} from 'react-native-ui-lib/testkit'`

Reply:
Huge thanks to @amitdahan for helping out with this one